### PR TITLE
Update a few docs documents after discussion

### PR DESCRIPTION
**Pull request by 0xGabi**
@bpierre I wonder why you decided to not include information about the App on the Role [here](https://github.com/aragon/plumbery/blob/master/docs/role.md)?

As I see it every `App` has a list of `Role`s that could be assigned `Permissions` on.

### CODE REVIEW

File: docs/connectors.md [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
 Three connectors are provided by default, listed below.
 
 ## ConnectorTheGraph
 
 A connector that fetches information from a The Graph subgraph.
 
+### Usage
+
+```js
+const connection = new AragonPlumbery({
```
Comment (bpierre) [inline docs/connectors.md]: Just to replace `connection` by `libraryName`:

```suggestion
const plumbery = new Plumbery({
```

File: docs/connectors.md [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+```
+
 ## ConnectorEthereumProvider
 
 A connector that fetches information from a standard Ethereum provider (EIP 1193).
 
+### Usage
+
+```js
+const connection = new AragonPlumbery({
```
Comment (bpierre) [inline docs/connectors.md]:
```suggestion
const plumbery = new Plumbery({
```

File: docs/plumbery.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ Note: “Plumbery” is a code name used during the development of this project
...
-| `config.connector` | `Connector | [String, Object] | String` | Accepts a `Connector` instance, and either a string or a tuple for embedded connectors and their config. |
-| `config.signer`    | `EthereumProvider`                      | An Ethereum provider (EIP 1193) used to sign and submit transactions.                                    |
+| Name               | Type                                      | Description                                                                                              |
+| ------------------ | ----------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| `config`           | `Object`                                  | The main configuration object                                                                            |
+| `config.connector` | `Connector \| [String, Object] \| String` | Accepts a `Connector` instance, and either a string or a tuple for embedded connectors and their config. |
```
Comment (bpierre) [inline docs/plumbery.md:12]: :100:

**Review by bpierre — COMMENTED** [review]
> @bpierre I wonder why you decided to not include information about the App on the Role here?

That’s a good point, roles wouldn’t be unique without their app so we should probably add it. I’m wondering if we should have an `app` property pointing to an `App`? Or if we should only have the `appAddress` to make it flat and compact.

> As I see it every App has a list of Roles that could be assigned Permissions on.

Yes I agree with that definition :+1:

**Comment by 0xGabi** [comment]
>I’m wondering if we should have an `app` property pointing to an `App`? Or if we should only have the `appAddress` to make it flat and compact.

That's a good point. I included the entities to have all available information as default. Maybe we can decide in which places make sense to flat the structure. I agree the `Role` is one of them.

**Pull request merged**